proposals/0001-template.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Extract WebView From Core
+author:
+- Jamon Holmgren
+date: 2018-07-30
+---
+
+# RFC0000: Extract WebView From Core
+
+## Summary
+
+Extract the current WebView implementation from React Native Core and move it into a third party package.
+
+## Basic example
+
+Currently, WebView is included in React Native core:
+
+```jsx
+import { WebView } from 'react-native';
+```
+
+This would move it to an external package, like so:
+
+```jsx
+import { WebView } from '@infinitered/react-native-webview';
```

Comment:
What should be the right source ? Should this be in infinitered, or should it be in react-native-community ?

Reply:
I'm open to react-native-community, but am more motivated to devote my company resources toward it if it lives in `infinitered`.

Reply:
packages get handed off as companies/individuals either lose interest in a tool, close their doors, no longer want to maintain the package itself, the primary maintainer moves to another organization, or a handful of other reasons. this has downstream consequences - for example, if react-native-maps was scoped under `@airbnb` on npm, we'd be stuck with that or have to force consumers to change their imports and package.json just because of a change in project maintainer structure.

I understand your concern about wanting to get some return for the time you invest, but I think there are better ways we can do this than brand the package name itself.

Reply:
@brentvatne Thanks for the feedback. I've removed the namespace and went with just `react-native-webview`.